### DISCUSSION TITLE
Greenlight / Breez LSP as Off-Chain Payout Processor

### DISCUSSION THREAD

**Discussion by svrnhdl** [btcpayserver/btcpayserver#6149] [Ideas / Feature Requests]
We've recently set up the Greenlight node, because we're getting tired of managing our own liquidity after all these years and there's just too many issues. It seems to work, but the payout processor for pull payments still uses the internal node. Are there any plans to enable the use of the LSP for this?

**Comment by Kukks** [comment]
It does not use the internal node. It uses the lightning node that is
configured with the store

**Comment by svrnhdl** [comment]
Aaah you are totally right! I had to change connection still. Worked now. :)
